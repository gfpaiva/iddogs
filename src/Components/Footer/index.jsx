import React from 'react';

export default function Footer() {
  return (
    <footer className="footer text-center">
      <p>
        Feito com
        <span role="img" aria-label="Coração">❤️</span>
        por:
        <a
          className="linked"
          href="https://guilherme.dev/"
          target="_blank"
          rel="noopener noreferrer"
        >
          Guilherme Paiva
        </a>
      </p>
    </footer>
  );
}
