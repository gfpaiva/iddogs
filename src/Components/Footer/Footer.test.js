import React from 'react';
import { mount } from 'enzyme';

import Footer from './index';

describe('<Footer />', () => {
  it('should mount properly', () => {
    const wrapper = mount(<Footer />);

    expect(wrapper.find('footer')).toMatchSnapshot();
  })
});
