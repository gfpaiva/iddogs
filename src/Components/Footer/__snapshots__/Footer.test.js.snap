// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should mount properly 1`] = `
<footer
  className="footer text-center"
>
  <p>
    Feito com
    <span
      aria-label="Coração"
      role="img"
    >
      ️ ❤️ 
    </span>
    por: 
    <a
      className="linked"
      href="https://guilherme.dev/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Guilherme Paiva
    </a>
  </p>
</footer>
`;
